cves/kernel/CVE-2020-15780.yml

```
@@ -75,7 +85,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: ["https://bugzilla.suse.com/show_bug.cgi?id=1173573"]
```

Write just the bug id.
```suggestion
bugs: [1173573]
```